File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        unordered_set<int>haha;
        int i=0;
        for(int num:nums){
        if(haha.count(num)==0){
        haha.insert(num);
        
        nums[i++]=num;
        }
        }
        return i;
        
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        # nums[:]=sorted(set(nums))
        # # return len(nums)
        if not  nums:
            return 0
        hey=1
        for i in range(1,len(nums)):
            if nums[i]!=nums[i-1]:
                nums[hey]=nums[i]
                hey+=1
                
        return hey
        